js/companies.js

```diff
@@ -17,40 +21,72 @@ var Companies = ( () => {
                 companyContainer[name] = new Company(name);
                 topBlocked.add(name);
             }
-            companyContainer[name].incrementCount();
+            companyContainer[name].increment('count');
             return companyContainer[name];
         },
 
+        /* addByPages is used by tab.js to count only unique 
+         * tracking networks on a tab
+         */
+        addByPages: (name) => {
+            if(!companyContainer[name]){
+                companyContainer[name] = new Company(name);
+                topBlocked.add(name);
+            }
+            if (name !== 'unknown') companyContainer[name].increment('pagesSeenOn');
+        },
```

Comment:
How about calling this `incrementPagesSeenOn (companyName)`. Otherwise this is really opaque to other people.